app/views/categories/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class= "nav-bar">
+  <div class="back-arrow">
+    <%= link_to categories_path do %>
+      <i class="fa-regular fa-arrow-left">Back</i>
+    <% end %>
+  </div>
+  <h2 class="login-head">NEW CATEGORY</h2>
+</div>
+<div class="notification">
+    <p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>
+    <div class="signout">
+        <%= button_to 'SIGN OUT', destroy_user_session_path, method: :delete %>
+    </div>
+</div>
+<div class="container">
+    <%= form_with model: category, method: :post, url: categories_path, local: true do |form| %>
+        <div class="field"> 
+            <%= form.text_field :name, class: 'form-control', placeholder: "Name" %>
+        </div>
+        <div class="field">
+            <%= form.text_field :icon, class: 'form-control', placeholder: "ICON URL" %> 
+        </div>
+        <div class="form-group form-submit">
+            <%= form.submit 'CREATE CATEGORY', class: 'btn btn-primary' %>
+        </div>
+    <% end %>
```

Comment:
- [ ] It looks like a turbo error occurs while trying to create a new category as shown below 👇 Please try to fix it
![image](https://user-images.githubusercontent.com/46839548/219818688-00eefc59-d728-4a54-9e7b-c3b61f5c1301.png)
You can reference this [resource](https://stackoverflow.com/questions/70400958/error-form-responses-must-redirect-to-another-location) to help fix the issue

Reply:
Hello @codepantha , Thanks for the review and feedback. However, I do not experience the same errors on my side. Both on the local and deployed versions of the app.

Please see [this](https://www.loom.com/share/ccf279ab65e7457e9f283db6a8c6778d) video of my chrome debugger when I add a new category

Reply:
Interesting! On closer inspection, I see that it only happens when the image link for the category looks like this  👇

```
data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAHwAAAB8...
```